bridge/bridge.go

```diff
@@ -181,15 +224,14 @@ func (b *Bridge) newService(port ServicePort, isgroup bool) *Service {
 	}
 
 	// not sure about this logic. kind of want to remove it.
-	hostname, err := os.Hostname()
-	if err != nil {
+	hostname := Hostname
+	if hostname == "" {
 		hostname = port.HostIP
-	} else {
-		if port.HostIP == "0.0.0.0" {
-			ip, err := net.ResolveIPAddr("ip", hostname)
```

Comment:
Why not just leave it blank if the docker container to be registered doesn't have any host IP binding? Resolving as the hostname of registrator container doesn't make sense to me, unless registrator is started via "--net host"...Leaving it blank works well in consul environment for us.